packages/frontend/src/test/knowledgeNuggets.test.ts

```diff
@@ -1,10 +1,7 @@
 import { KnowledgeNugget, layer2s } from '@l2beat/config'
 import { expect } from 'earljs'
 
-const knowledgeNuggets = layer2s.reduce<KnowledgeNugget[]>((nuggets, l2) => {
-  nuggets.push(...(l2.knowledgeNuggets ?? []))
-  return nuggets
-}, [])
+const knowledgeNuggets = layer2s.flatMap<KnowledgeNugget>(nugget => nugget.knowledgeNuggets ?? [])
```

Comment:
is `<KnowledgeNugget>` type needed here? also formatting failed

Reply:
fixed formatting, KnowledgeNugget is needed indeed

Reply:
Are you sure? I just deleted it locally, and everything still works. It's inferred because `layer2s` is typed properly.